testing/tests/consensus.rs

```diff
@@ -62,30 +62,13 @@ enum QueuedMessageTense {
 /// Returns true if `node_id` is the leader of `view_number`
 async fn is_upcoming_validating_leader<
     TYPES: NodeType<ConsensusType = ValidatingConsensus>,
-    I: NodeImplementation<TYPES, Leaf = ValidatingLeaf<TYPES>>,
+    I: TestableNodeImplementation<TYPES> + NodeImplementation<TYPES, Leaf = ValidatingLeaf<TYPES>>,
```

Comment:
Could we simplify this to
```
I: TestableNodeImplementation<TYPES, Leaf = ValidatingLeaf<TYPES>>
```
since `TestableNodeImplementation` already implies it implements `NodeImplementation`?

Reply:
Switched!